Config/BaseFlow.ini

```diff
@@ -1,5 +1,6 @@
 [CoreRedirects]
 +ClassRedirects=(OldName="/Script/Flow.FlowNode_CustomEvent",NewName="/Script/Flow.FlowNode_CustomInput")
 +PropertyRedirects=(OldName="FlowAsset.CustomEvents",NewName="CustomInputs")
++PropertyRedirects=(OldName="FlowGraphNode.FlowNode",NewName="NodeInstance")
```

Comment:
Renamed this member to make the conversion for FlowGraphNode operating on FlowNodeBase easier and clearer, when casting to FlowNodes.

It also has a precedent in a lot of the UE UEdGraphNodes